diff --git a/WorkflowView.vue_vue_type_style_index_0_lang-Bo5xJraI.mjs b/WorkflowView.vue_vue_type_style_index_0_lang-Bo5xJraI.mjs
index 3a5b5c3630b7e491c5b913e76a28f0a66cc0c4b8..897eec6e3606b646e40fa10412ba9a900ee17fee 100644
--- a/WorkflowView.vue_vue_type_style_index_0_lang-Bo5xJraI.mjs
+++ b/WorkflowView.vue_vue_type_style_index_0_lang-Bo5xJraI.mjs
@@ -1192,7 +1192,7 @@ function _n(o, t, {
   const { properties: n } = t, { options: i } = n, s = [
     ...e({ text: "输入变量", required: !1 })
   ];
-  return i.model.modeId || s.push("必须选择模型"), s;
+  return i.model?.modeId || s.push("必须选择模型"), s;
 }
 function kn(o) {
   const { mergeIOParams: t, updateOption: e } = nt(o, We);
